lib/src/utils/crypto-utils.ts

```diff
@@ -154,8 +153,7 @@ export class CryptoUtils {
      */
     public static decodeIDToken(idToken: string): DecodedIDTokenPayload {
         try {
-            const words = Base64.parse(idToken.split(".")[1]);
-            const utf8String = utf8.stringify(words);
+            const utf8String = base64url.decode(idToken.split(".")[1], "utf8");
```

Comment:
WDYT about checking if there is an element at position `1` to make the operation fail safe?